### ISSUE TITLE
Media thumbnail tombstone standardization

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#2813]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Describe the bug**
Pre-PG migration media description would show in the catalog record as well as at media details. Now to see this important field you must click edit media. This label (not sure what other labels are not appearing - made date and checksum seem to show) often contains vital information for our users, who are incapable of seeing this content now.

**Expected behavior**
Media label description must be visible at media details at minimum, and in the thumbnail summary on the catalog record ideally.
See any UAM:EH records with media for examples, but my screenshots come from http://arctos.database.museum/guid/UAM:EH:0789-0001

**Screenshots**
Catalog record view:
<img width="466" alt="Screen Shot 2020-06-20 at 8 37 45 AM" src="https://user-images.githubusercontent.com/17605945/85206868-8e16f680-b2d1-11ea-9134-d41845f60a19.png">

Media details view (operator view):
<img width="613" alt="Screen Shot 2020-06-20 at 8 37 57 AM" src="https://user-images.githubusercontent.com/17605945/85206882-9d963f80-b2d1-11ea-9ce7-25797b98220d.png">

Edit media view:
<img width="1056" alt="Screen Shot 2020-06-20 at 8 38 20 AM" src="https://user-images.githubusercontent.com/17605945/85206896-b141a600-b2d1-11ea-8dd4-31bf4c2e8e71.png">

Media detail page, public user view:
<img width="654" alt="Screen Shot 2020-06-20 at 8 38 45 AM" src="https://user-images.githubusercontent.com/17605945/85206909-bf8fc200-b2d1-11ea-80c0-0569de41aa41.png">

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Mac OS10.14.6
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] 83.0.4103.97

**Priority**
Critical - vital media usage/credit information is blocked from users.

**Comment by krgomez** [comment]
I have noticed this as well and it seems to me that labels display below media on catalog records is even more random than it was before. For all our media I used the description label to record the copyright notice to accompany the media. Previously this would _almost_ always display below media (though sometimes it would randomly not display). We also recorded our photo credits as descriptions, and for some reason it was always the copyright notice that would display instead, which is precisely what we want. I figured this was because of the © symbol -- I never really got an answer on this topic when I brought it up on GitHub before. Anyway, now what I've been noticing is that our photo credits are displaying on some records instead of the copyright notice that used to display. I have also noticed the same thing that Angie shows above, where no descriptions are displaying in the media details page, though sometimes they are visible there. It all feels quite random. I would much prefer that we can predict what will display below the media on catalog records and as Angie mentions, at a minimum, we need this information to display on the media details page.

**Comment by Jegelewicz** [comment]
See also #1930

**Comment by dustymc** [comment]
>Media label description must be visible at media details 

Done at test.

> in the thumbnail summary

It might be easiest to start from scratch there - what do you want that to look like? 

Input can be about anything (no description but useful relationships, 500 4000-character descriptions, anything!). Output needs to fit in a compact grid, unless we're rebuilding that as well.

>used the description label to record the copyright notice to accompany the media.

That seems destined to cause problems. Why not request a new value in https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label?

**Comment by krgomez** [comment]
> That seems destined to cause problems. Why not request a new value in https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label?

If we were to add a copyright notice media label, would it be possible to somehow move all the notices I've already added as media descriptions to the new label?

**Comment by dustymc** [comment]
>possible to somehow move all the notices

Moving is easy, finding them mixed in with whatever else is there probably isn't. I'm certainly happy to do what I can.

**Comment by Jegelewicz** [comment]
See also #2529

**Comment by dustymc** [comment]
Thanks @Jegelewicz.

If there are any more Issues that might change the "tombstone" (or thumbnail view in general), can someone round them all up into one uber-issue? 

Then I suggest we...

1. clean up existing labels if necessary, make sure we have whatever new labels we need, particularly those used in the thumb-view. (I'm not sure I see the split between description, title, and subject - minimally those need better documentation.)
2. then discuss what/how gets displayed

**Comment by Jegelewicz** [comment]
@krgomez if you can mock up what you want, I'll help usher it through. @AJLinn will probably have something to say about it too.

**Comment by Jegelewicz** [comment]
Also #1652

**Comment by dustymc** [comment]
>mock up

That's sounds like a good place to start, but we also need to deal with

* has a bunch of whatever "fields" ya'll want
* has what someone considers "title" information in some "field" that ya'll don't use
* just has relationships (any or all of https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_relationship, repeated any number of times) from which we might pull something
* a bunch of stuff I haven't thought of

Or maybe that's just too complex for anyone to deal with and we need an explicit "display this in the thumbnail" label (which we could populate with all ^^that^^ junk). That would be a lot more work to maintain - you might need to change it when you re-identify a linked item, which will never happen - but it would take all the mystery out of this, and could even be constrained to a certain number of characters or similar.

**Comment by Jegelewicz** [comment]
What about something in manage collection, so that "display this with media thumbs" could be standard by collection? That way you don't have to do it for every media item you create.

**Comment by dustymc** [comment]
There is no relationship between media and collection.

**Comment by Jegelewicz** [comment]
Dang it! But could the display of media on a catalog record page be modified by collection? So that all thumbs displaying within a UAM:Art page would display labels x,y and z, but thumbs on a NMMNH:Paleo record would display labels x, y and a?

Of course that is just getting into - "go make your own page display" territory....

**Comment by dustymc** [comment]
That seems technically plausible, but I think at best it would be incomplete, and at worst it would be confusing - a user could see the same image formatted 20 different ways in 20 different views.

Its also more complexity and resource requirements than I'd like to introduce, but that's another discussion.

**Comment by campmlc** [comment]
Is it possible to make some connection between media and collection? This
seems especially relevant in terms of media licensing, which may be both
collection-specific and object specific.

**Comment by dustymc** [comment]
Sure, but it's Major Surgery (and probably causes a whole bunch of problems that would need solved).

> media licensing, which may be both collection-specific and object specific.

Also major surgery?? I'm not sure what you mean but this is probably a good place to address it.

**Comment by krgomez** [comment]
Do you guys want me to start a new issue - broadly about media labels? My thoughts right now: 

-Need for consistent display of a designated media field below thumbnails, whatever field that may be. For us, I'm not sure if it should be a copyright notice or some kind of aggregated tombstone/citation field, including copyright notice.
-Maybe new label to add: copyright notice. For copyright, I think there would be a need to be able to link to media from a future Arctos rights module.
-For the photo credits I added to many media as description labels, I wonder if it would be better to actually add the person credited to the media record as an agent. It seems like it would need to be a different relationship than "created by". Sometimes the person who photographed/digitized an object/whatever is not the same person entering that into Arctos. I think there are cases where there wouldn't be a need to record who scanned a document for example (at least I think that's not something one would specifically record -- maybe I'm wrong), but we do in many cases like to record a credit of who photographed an object.
-We have not really used other labels so far, at least for our collection images. I have used some other labels for linking to oral history audio recordings and I think I found it less than straightforward knowing which labels to use to reference external media.
-Media made date is potentially confusing? In some cases it would be relevant to add in the date the original media was created (for example, oral history recording date) as well as the date the media was created in Arctos. Maybe there is already a way to document this that I'm not aware of.

**Comment by dustymc** [comment]
>issue

Yes please, or just take over this one - I'm never sure which is better.

>designated media field below thumbnails

A "this displays in thumbs" "field" is simple and predictable (and the data could be controlled) - sounds fabulous to me, if it works for users.

>future Arctos rights module.

That's another discussion, but the vague idea is that it'll go round up any sort of "rights" data (and then cause display problems...) - pulling a label and a license (and whatever) seems well within scope.

>better to actually add the person 

100000% yes

>photographed/digitized an object/whatever 

There's some judgement there. If it's a real photographer, sure. If it's the student who shoved 200000 plants under the copy stand, or me firing off scripts at 20TB of binaries, I'm not sure recording that is anything but distracting. Just a matter of documentation, probably.


>which labels to use to reference external media.

None - labels are for text, referencing ANYTHING is the job of https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_relationship

(And Arctos doesn't care where the Media are stored, if that's what you mean by "external.")

>made date ...  oral history recording date

yes, that's the idea

>media was created in Arctos

That's an easy job for a trigger. We don't have that for media (maybe there's an issue somewhere??) but we do for labels and relationships.

```
arctosprod@arctos>> \d media_relations
                                            Table "public.media_relations"
       Column        |            Type             | Collation | Nullable |                  Default                   
---------------------+-----------------------------+-----------+----------+--------------------------------------------
 media_relations_id  | bigint                      |           | not null | nextval('sq_media_relations_id'::regclass)
 media_id            | bigint                      |           | not null | 
 media_relationship  | character varying(40)       |           | not null | 
 created_by_agent_id | bigint                      |           | not null | getagentidfromlogin(SESSION_USER::text)
 related_primary_key | bigint                      |           | not null | 
 created_on_date     | timestamp without time zone |           |          | LOCALTIMESTAMP

```

**Comment by Jegelewicz** [comment]
> 
> 
>     issue
> 
> Yes please, or just take over this one - I'm never sure which is better.

I say just take over this one - we can rename the issue.

>     designated media field below thumbnails
> 
> A "this displays in thumbs" "field" is simple and predictable (and the data could be controlled) - sounds fabulous to me, if it works for users.

I like the idea of selecting no more than 3 fields: copyright, description, ? but we need to make it clear that if you add more than one description, only one is gonna show and it might not be the one you want, OR we need to add a term "thumbnail description".

>     future Arctos rights module.
> 
> That's another discussion, but the vague idea is that it'll go round up any sort of "rights" data (and then cause display problems...) - pulling a label and a license (and whatever) seems well within scope.

Agree - let's save this for later

>     better to actually add the person
> 
> 100000% yes

YES - do we need a new field or can we make use of "created by agent"?

>     photographed/digitized an object/whatever
> 
> There's some judgement there. If it's a real photographer, sure. If it's the student who shoved 200000 plants under the copy stand, or me firing off scripts at 20TB of binaries, I'm not sure recording that is anything but distracting. Just a matter of documentation, probably.


Why? They still took the photo - I don't see a distinction here. Does being an artist only make you a "real" photographer? NO

>     which labels to use to reference external media.
> 
> None - labels are for text, referencing ANYTHING is the job of https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_relationship
> 
> (And Arctos doesn't care where the Media are stored, if that's what you mean by "external.")

Maybe this refers to media that is not "linkable"? Do barcodes need extended here? As in physical photo is in whatever photo album in wherever place?

>     made date ... oral history recording date
> 
> yes, that's the idea
> 
>     media was created in Arctos
> 
> That's an easy job for a trigger. We don't have that for media (maybe there's an issue somewhere??) but we do for labels and relationships.

I think we should add this - in the same way we have "created by" in every object record, we should have "created by" as well as "last edited by" for media. We should also have an edit history somewhere.....

**Comment by dustymc** [comment]
>Why? They still took the photo

For me, it makes search difficult. I've got a couple hundred photos of rat teeth and deal whales and such that I might have reason to find, and several hundred thousand where I was involved in some sort of near-automation that I probably won't. I'm still not suggesting hard rules, but **I** would prefer not to be "creator" for those. 

>Do barcodes need extended here?

Media label barcode was introduced for that (mostly during processing) - part Media can formalize the link.

>"created by" 

That's easy enough, but it'll be "DBA" for most images.

>edit history

This carries the same issues as editing anything else in a complex system. It's likely actually plausible under the current infrastructure, but it would be a lot of work and require significantly more resources than we have (I'd guess dozens of terabytes per year). There's a giant issue out there somewhere.

**Comment by dustymc** [comment]
Dang I missed the important part...

>I like the idea of selecting no more than 3 fields: copyright, description, ? but we need to make it clear that if you add more than one description, only one is gonna show and it might not be the one you want, OR we need to add a term "thumbnail description".

>copyright

There are currently three things below media.

* filetype
* license
* text

The rights module will (presumably) gather license and copyright - I think it SHOULD NOT be in the text, or it'll display twice.

>only one is gonna show

Or only SOME of one. Those are currently 4K characters, which is now an imposed limit - we could bump them up to whatever PG's actual limit is and this could be gigabytes of data. That's not going to display, and it's probably not something we want to try to shove behind a mouseover (where it can eat browsers and networks) or something. If we can live without the 'try to magic something out of relationships' functionality, I really like the idea of a dedicated 'thumbnail description' concept.

**Comment by krgomez** [comment]
> The rights module will (presumably) gather license and copyright - I think it SHOULD  NOT be in the text, or it'll display twice.

So you are saying that if we already have the copyright notice recorded in the future rights module, that it could then be pulled into the media labels somehow? It sounds like you are leaning toward a thumbnail description field. We love the idea of a citation/attribution/tombstone caption below the thumbnail. The reason that it would be great to display this caption below the thumbnail is that for users who may use images of works in the collection by fair use and not go through a formal image use request, providing them with the attribution that we want them to use would be ideal. However it seems that the characters should not be limited so that the entire caption displays.

Sounds like if a photographer/digitizer needs to be credited, that this should be an agent relationship. Sometimes we may want to credit the specific photographer, however the source that we want to credit for most of our collection images is UAMN. The individual who photographed/digitized an object is different from the holder of the media. Maybe roles involved in media need to be clarified through different agent relationships. Also, the source/photo credit may potentially be part of attribution/citation information, and so does it make sense to also use text in order to flexibly record any complexity that there may be or specific language that is required to be cited?

> "created by"

Are you saying we would create a new agent relationship label that is like "created by" or just use "created by" for various roles?

**Comment by dustymc** [comment]
>notice recorded in the future rights module

I was thinking you'd have the existing label which would get pulled into the rights module (which would just be code, not any data of it's own), but that's something we can work out there.

>leaning toward a thumbnail description field

I have no opinion. That's simple (easy for me) and predictable (I think what you're looking for), but also not capable of summarizing relationships and such. I don't know how others would feel about that; we probably need a new issue to find out if that starts to look like the best path from here.

>UAMN

That's just another agent - https://arctos.database.museum/agent.cfm?agent_id=3691&agent_name=University%20of%20Alaska%20Museum%20of%20the%20North

> also use text 

Sure, if you find it useful/necessary. 

> new agent relationship label that is like "created by" or just use "created by"

Also a new Issue - I don't care as long as whatever we do it documented and doesn't change the meaning of any existing data.

**Comment by krgomez** [comment]
Okay, I will start a thumbnail description label issue. 

For all of the collection images that I have uploaded for the art collection, I used myself as the "created by" agent, as it was automatically added when individually uploading the media and I think I assumed that it referred to created in Arctos. I don’t think we really considered the role that the “created by” agent has in media several years ago when I began uploading all our media and I’m not sure I had ever looked at the documentation for that label, which says: "Person who pushed the button on a camera. Not the person who e.g., scripted a million JPGs (nobody cares). Related term must be an exact match for a unique agent name." For some of the media I uploaded I did actually take the photos, for most of them I did not take the photos but was involved in processing of the image files, and for all media I resized files for Arctos. I think we want to change the “created by” agent from Karinna Gomez to UAMN. This would better reflect the source of the media. In terms of other new agent roles for media, it seems a little complicated and it probably needs some discussion. We started this table to help us think through some of this complexity: https://docs.google.com/document/d/1CSqDLriZZvOtPFbD4QxbIPKDj8T8JaJk6EntDUrpVHw/edit?usp=sharing

> I was thinking you'd have the existing label which would get pulled into the rights module (which would just be code, not any data of it's own), but that's something we can work out there.

I have added the copyright notice for any one artwork in two places (I realize we should probably only record it in one place, which would ideally then display in multiple places, wherever it needs to be seen -- hopefully the rights module will help with this). In media, I have added the copyright notices as description labels. In the catalog records, I have recorded the copyright notices as remarks for the copyright status attribute. As a side note, is it possible to add that remark as a column option for display in search results and download?

**Comment by campmlc** [comment]
As a related item, allowing attribute remarks for download as a separate
column would also help parasitology, especially, since critical info is
stored here.

**Comment by dustymc** [comment]
>is it possible to add that remark as a column option for display in search results and download

The short answer is no, because there is no column. There is cardinality involved - if you can create one attribute, then you can also create 54789 of them, and it becomes difficult to pretend that Arctos is a giant spreadsheet under those conditions.

The longer answer is that we can probably fake it in some way you'd find acceptable, but that needs a dedicated issue in which I'll need to know what form of munging multiple values into a "field" you'd find acceptable.

(The correct answer may involve normalization, but I'm not sure what you're attempting to do so I'm not sure about that.)

**Comment by krgomez** [comment]
Maybe our copyright notices can live somewhere else once a rights module exists? I think we need a way to download copyright notices. The core data ("tombstone") for artworks includes: artist/creator, title, creation date, medium, dimensions, credit line, and copyright notice. The copyright notice wouldn't appear on an exhibition label but it would appear in a publication or other places. We need to be able to download the core data for the collection. Right now we can download all of those data except the copyright notice.

**Comment by dustymc** [comment]
What does the notice apply to?

**Comment by krgomez** [comment]
The notice applies directly to the physical object/work and reproductions of that object/work (media). The artist/creator is usually the copyright holder and the notice format is basically © artist or artist's estate. The UAM:Art collection is made up of mostly 20th/21st century art and almost all of it is under copyright. We have very few public domain works.

**Comment by krgomez** [comment]
Maybe we need a new free-text attribute: copyright notice?

**Comment by dustymc** [comment]
Thanks, that helps.

In that case I think repeating that information may NOT be denormalization, but just a necessary reflection of what the data applies to.

* An attribute covers the physical thing represented by the data
* A media label covers the derivative, which may or may not carry the same copyright as the original (eg, maybe the photographer has a claim, or "its complicated," or etc.)

If that's correct then yes, I think a dedicated attribute makes sense.

**Comment by krgomez** [comment]
Yes, this summary is correct. Should I make an issue for a new attribute? Will it be possible for us to "move" the copyright status remarks to a new attribute?

**Comment by Jegelewicz** [comment]
@krgomez yes and yes - assuming it will be easy to separate the copyright status from anything else in the remarks.

**Comment by Jegelewicz** [comment]
@krgomez are we done here or do we need to kick these tires?

**Comment by krgomez** [comment]
@Jegelewicz To my knowledge this isn't resolved yet, so I would rather not close the issue. I'm not sure what else to add to the discussion at this point though.

**Comment by Jegelewicz** [comment]
@krgomez OK. I will try to summarize and see what we need to do.

**Comment by krgomez** [comment]
@Jegelewicz I just went back through this whole issue again. I had created #2846 proposing a new media label _thumbnail description_ but the issue was closed and "merged" with this one. Above, you and Dusty both said to just take this one over instead of creating a new one, so I have no idea why I made a new issue. This is what I proposed there:

Goal
Describe what you're trying to accomplish.
We need a designated media field that will display below thumbnails on catalog records. See #2813, #1652, #2529.

Context
Describe why this new value is necessary and existing values are not.
Currently, media labels (description, comment, and maybe others) unpredictably display below thumbnails. We need the ability to predictably display critical media data to users on catalog records. By calling this media label “thumbnail description”, it allows flexibility for collections to prioritize what type of data they consider to be most important to display.

Table
Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm
media_label

Value
Proposed new value
thumbnail description

Definition
Clear, complete, non-collection-type-specific definition of the new value.
Description that will display below thumbnails on catalog records.

**Comment by dustymc** [comment]
>thumbnail description

No objection, but I sorta like "tombstone."

>Description that will display below thumbnails on catalog records.

That can't work, for two reasons.

1. Most Media aren't likely to get this label. Ya'll can tell me what you want to see, otherwise it'll be something like
```
IF thumbnail description THEN
   use that
ELSE
  use description
```
and the documentation should reflect that.

2. I don't/can't know/care where the media view is invoked from - whatever you see in catalog records will also be what you see in taxonomy and locality and the bajillion other places media are or might become visible, which should also be reflected in the definition.

There will also be length and cardinality limitations - 500000 characters isn't going to fit in any current (or probably future) UI, nor will 5000 "thumbnail description" labels no matter the contents. I don't have anything hard and fast to offer, but "add only one of these" and "keep it concise" might be useful things to include in the definition as well.

**Comment by krgomez** [comment]
> tombstone

I also like tombstone, but perhaps it's too weird/specific?

> IF thumbnail description THEN
   use that
ELSE
  use description

Sounds good to me.

> There will also be length and cardinality limitations - 500000 characters isn't going to fit in any current (or probably future) UI, nor will 5000 "thumbnail description" labels no matter the contents. I don't have anything hard and fast to offer, but "add only one of these" and "keep it concise" might be useful things to include in the definition as well.

For what I'm picturing, 400 characters would probably suffice for our use of this media label. I think most of our captions would be more like 200-300 characters. Would the text fully display below the thumbnail or would it be truncated and only fully visible within media details?

**Comment by dustymc** [comment]
>weird/specific

Weird maybe, but it's a nice word for this. This _is_ something very specific, so I see that as a good thing. Still has my vote, whatever that's worth.

>truncated

I don't have a great answer, there are some adjustments which could be made in the current UI (font size, spacing, etc.), I doubt it would be too complicated to even replace the current UI, but 400 seems like a lot to stuff into an environment that might be showing hundreds of images. Probably better discussed after you've got some data, just something to be aware of.

**Comment by Jegelewicz** [comment]
Thumbs up to "tombstone" and the if/else thingee.

as for the size - I think it is going to be a problem - maybe we could test this in test?

**Comment by dustymc** [comment]
>test this in test?

This will just replace description when it exists, there's not much new.

And media-stuff is cached and I can't run the maintenance scripts in test because https://github.com/ArctosDB/internal/issues/65, which maybe doesn't matter here but will if this turns into messing with the layout.

**Comment by Jegelewicz** [comment]
> test this in test

Yeah I was thinking of the project with 400 images.

**Comment by Jegelewicz** [comment]
@dustymc do we need to clarify anything here? Do I just need to create the new term - tombstone - in [media_label](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label)? @krgomez anything to add?

**Comment by krgomez** [comment]
I don't think so! It would be great to have this new media label.

**Comment by Jegelewicz** [comment]
OK - created a new issue for the code table add. Once that is done, we can return here for implementation in UI.

**Comment by dustymc** [comment]
I'm no longer sure what we're doing here, if I ever was.

One clear "this be the tombstone" label seems like an improvement, but that would need accompanied by some migration path or documentation or something.

One more item for the "if available use else try next" blackbox pile of potential tombstone content seems like just a bit more of the confusion that lead here.

@krgomez can you explain how this isn't just another way of spelling [description](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label#description)?

**Comment by krgomez** [comment]
If we could reliably say that the description would always display below the thumbnail, then it seems fine to continue using it and not creating a new label. We need:
-- To know what label will display below the thumbnail and that it will reliably do so. I don't think it matters what this label is called, whether it be description, tombstone, thumbnail description, citation, etc., as long as we know what label will be prioritized to display.
-- To have a label that can be used to record a citation: https://github.com/ArctosDB/arctos/issues/2529#issue-572178528

Because attribution is essential for image use, the label we ideally want to display is the tombstone, which could basically be considered a citation. I like the idea of making it easy for a user to find the attribution for their image use by having it displayed right along with the thumbnail. Because of the UI of Arctos, the elements of the tombstone are located in different parts of the catalog record, so having them prepared together in one spot would be helpful.

However, if it is not possible to display a label that has up to about 200-300 characters (tombstone length), then it could be recorded as a citation/tombstone label that does not display below the thumbnail, but would show on the media details page. In this case I would then want the copyright notice to display below the thumbnail, which is basically what's happening now. However if are any other description labels recorded for the media, it's uncertain which will display, which has been a problem for us.

**Comment by dustymc** [comment]
Thanks!

Sorta high-level, I wonder if just finding a way to get better tools (https://github.com/ArctosDB/arctos/discussions/4343) shouldn't be seen as a sort of prerequisite - this seems like something the IIIF crowd might have solved, anyway. 

I don't know that I can do anything very radical with the "thumbnail view" given the tools I have, 300 characters of text would certainly require a hard rethink of basically the whole thing (and it's everywhere).

Having things like more-predictable thumbs is not completely unrelated to the media detail page, but I do have a lot more flexibility there. Probably another Issue, but radical redesigns don't seem too unrealistic, and shuffling some things around might even tend towards trivial.

>having them prepared

I thought that Arctos rounding up various things was the intent of https://github.com/ArctosDB/arctos/discussions/4972 (or maybe some other discussion that became that??), but maybe I just didn't understand something or that's a separate conversation?

Or maybe you're suggesting that people need to round up whatever's relevant and condense it into the tombstone??

Would be good to get this sorted out before https://github.com/ArctosDB/arctos/issues/3331, and to use that to implement whatever we come up with.

I become more and more a fan of simplicity as Arctos gets more complex and diverse, so I'll tentatively advocate for one human-supplied label ([description](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label#description) seems as good as anything), and whatever tools/updates/whatever are needed to make that work.

**Comment by krgomez** [comment]
I think that if it is not possible to display more characters for the label that appears below the thumbnail, then I would say we should just keep the description as the label that displays and I will keep using it to record a copyright notice if there is one. Is it the only label that does display or are there others?

I still think it would be nice to have a label where we can record the tombstone/citation caption though, and based on #2529 it seems there is a need for something similar for other collections. I do think that it would be more user-friendly for the public to easily find the caption we prefer them to use for attribution without having to click media details, because I just don't think people will do that.

A tombstone caption would be different than what we're talking about in #4972. A tombstone caption for an artwork would generally include: artist name, artwork title, year created, media, dimensions, credit line, catalog number, copyright notice. It's the core identifying information about an artwork. I would be able to make these pretty easily for everything since each of these pieces of information can be downloaded.

If we keep description as the label that displays and I have two descriptions for many media, is there a way to remove one of those descriptions? I have one for a copyright notice and one for a photo credit. The photo credits could instead be comments or some other suitable label. Right now it is unpredictable which of the two descriptions will display.

**Comment by Jegelewicz** [comment]
@krgomez @dustymc I am trying to get to some older issues to see if we can make any progress.

> A tombstone caption would be different than what we're talking about in https://github.com/ArctosDB/arctos/discussions/4972. A tombstone caption for an artwork would generally include: artist name, artwork title, year created, media, dimensions, credit line, catalog number, copyright notice. It's the core identifying information about an artwork. I would be able to make these pretty easily for everything since each of these pieces of information can be downloaded.

How about a simpler, smaller tombstone like those on the [Getty](https://www.getty.edu) site?

![image](https://user-images.githubusercontent.com/5725767/219436644-b329ea1b-cbb6-4cdc-a03e-f58c75702800.png)

![image](https://user-images.githubusercontent.com/5725767/219432021-ac177a28-b3b4-4981-a294-b43b1f3bbdaa.png)

This appears to include
[title (description)](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label#description)
[artist (created_by_agent)](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_relationship#created_by_agent)

Can we build that "tombstone" and would it do? Clicking the image should take one to all of the image details:

https://www.getty.edu/art/collection/object/109PDZ including a [link to the catalog record](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_relationship#shows_cataloged_item)

https://www.getty.edu/art/collection/object/109PDZ#full-artwork-details

Issues I can see here:

Extremely long descriptions would be truncated and [description](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label#description) can be used more than once, so things could get nutty. I don't think biological collections would care, but art and ethnology collections would. Perhaps we need a very specific label for `artwork title` since [title](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label#title) is already being used for something else.

If this seems like a good solution, then we can talk about how a media detail page should look.

![image](https://user-images.githubusercontent.com/5725767/219438852-8c2686ef-65ef-4770-809c-1a3475cab366.png)

as compared to

![image](https://user-images.githubusercontent.com/5725767/219439764-22137701-b088-4aaf-8d17-1b9625c9cdf5.png)

**Comment by krgomez** [comment]
One thing I did since we last discussed this issue was I fixed all of our media under copyright to only have one description, which shows the copyright notice and displays consistently below the media thumbnails. However maybe it would be preferable to use a label that includes the artist's name and the work's title, particularly since some works (like the Henry Wood Elliott watercolor above) are in the public domain and do not require a copyright notice. I like the idea!

**Comment by Jegelewicz** [comment]
I also see that created_by_agent is not the artist, so we would need something new there? So I guess we would need a new relationship. Maybe something like

maker - the agent who made the item depicted in the media

where the name is linked to the agents table

BUT that will also run into the verbatim agent problem! Any works with only a verbatim agent would not display the artist name, or could only display "undetermined" if we end up with the solution I proposed in https://github.com/ArctosDB/arctos/issues/5576#issuecomment-1432001508

**Comment by krgomez** [comment]
Yes, the created_by_agent for all our media is me. I had a hand in processing all of our media and uploaded it all, so that's why I'm the agent there. I think there was maybe an issue about using entered_by_agent instead? I can't seem to find it... but I do think that would be a better description of what I did with the media, as created_by_agent could then be used for the artists. If there's no problem with leaving my name where it is in the media metadata and adding a new label linking the artists to the media, that would work. Not sure what to say about the verbatim agent problem! Maybe there would be a way to manually fix those in the media metadata?

**Comment by dustymc** [comment]
I am tabling this; I think it needs attention and probably https://github.com/ArctosDB/arctos/discussions/4343 to be reopened.
